force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +62,10 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 != null && num2 != null) {
+            largestNum = Math.max(num1, num2);
+        }
```

Comment:
generally, we check for invalid values or arguments first: this is called "returning early"
```suggestion
        if (num1 == null || num2 == null) {
            // return null or your error value or throw an exception
        }
        // now we do our logic
        return Math.max(num1, num2);
```